lib/routes/account.js

```diff
@@ -390,6 +395,26 @@ module.exports = function (
             )
         }
 
+        function checkNumberOfActiveSessions () {
+          return db.sessions(emailRecord.uid)
```

Comment:
Oh I see, I did fail to notice that the `sessions` variable here is used further below.